document/CoreFormat.mdk

```diff
@@ -2439,9 +2440,35 @@ C++: `"std::runtime_error"`
 C#: `"System.ArgumentNullException"`
 ~
 
+If the tool does not have access to information about the object that was thrown, the `type` property shall be absent.
+
+~ IsoNote
+This will be the case in C++ programs that throw objects that do not derive from `std::exception`:
```

Comment:
`std::exception` has no way to ask for the type. In C++'s case the best that can be done is is probably going to be something like "Unhandled C++ Exception" or similar.

(Recovering the dynamic type of the exception is possible but generally requires a debugger / debugging symbols)